### PR TITLE
wip

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-52b73bda9b/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-52b73bda9b/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-70039377c7/how-to/)

File: content/managed-operations/rackspace-ssl-certificates.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+To purchase an SSL certificate from Rackspace, take the following actions:
+
+1. Open the Customer Portal and click on **Network**.
+2. Select **SSL Certificate**.
+3. Click on **Actions** in the upper-right corner.
+4. Select **Purchase Certificate**.
+5. Choose the certificate you want from the list.
+
+After you have purchased a certificate, a ticket is generated, which provides
```
Comment (RackerKate) [inline content/managed-operations/rackspace-ssl-certificates.md]: Please clarify what the ticket does. Is it correct to say that "When you purchase a certificate, the system generates a support ticket to install the certificate," or does the "ticket" simply give the user info on how to install it themselves?

File: content/managed-operations/rackspace-ssl-certificates.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+the process for certificate installation.
+
+To learn more about the type of certificates, click on **Compare SSL
```
Comment (RackerKate) [inline content/managed-operations/rackspace-ssl-certificates.md]: Suggest "...types of certificates..."

File: content/managed-operations/rackspace-ssl-certificates.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+Certificate** in the **Certificate Type** section to see a comparison chart that
+shows the following information:
+
+- Pricing by SSL type and duration
+- Details for the SSL, which you can see by clicking on the SSL column heading
```
Comment (RackerKate) [inline content/managed-operations/rackspace-ssl-certificates.md]: Bold the 2nd SSL in this sentence?

File: content/managed-operations/rackspace-ssl-certificates.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+- Feature comparison by SSL type
+
+### After you have a certificate
+
+For help installing your new ssl certificate, see [Install an SSL certificate
```
Comment (RackerKate) [inline content/managed-operations/rackspace-ssl-certificates.md]: Capitalize SSL.

File: content/managed-operations/rackspace-ssl-certificates.md (hunk 1 above)
Comment (catlook) [reply content/managed-operations/rackspace-ssl-certificates.md]: it provides the instructions

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6737cdece7/how-to/)

File: content/managed-operations/rackspace-ssl-certificates.md (hunk 1 above)
Comment (RackerKate) [reply content/managed-operations/rackspace-ssl-certificates.md]: In that case, suggest something like "After you purchase a certificate, the system generates a ticket that gives you the instructions for installing the certificate."

File: content/managed-operations/rackspace-ssl-certificates.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+shows the following information:
+
+- Pricing by SSL type and duration
+- Details for the SSL, which you can see by clicking on the **SSL** column heading
+- Feature comparison by SSL type
+
+### After you have a certificate
+
+For help installing your new SSL certificate, see [Install an SSL certificate
+on Apache](/how-to/installing-an-ssl-certificate-on-apache/)
```
Comment (RackerKate) [inline content/managed-operations/rackspace-ssl-certificates.md]: Add period at end of sentence.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-407074f7b0/how-to/)

**Pull request merged**